test/rekt/features/apiserversource/data_plane.go

```diff
@@ -351,6 +352,151 @@ func SendsEventsForAllResources() *feature.Feature {
 	return f
 }
 
+func SendsEventsForAllResourcesWithNamespaceSelector() *feature.Feature {
+	source := feature.MakeRandomK8sName("apiserversource")
+	sink := feature.MakeRandomK8sName("sink")
+	f := feature.NewFeatureNamed("Send events for all resources within select namespaces")
+
+	f.Setup("install sink", eventshub.Install(sink, eventshub.StartReceiver))
+
+	sacmName := feature.MakeRandomK8sName("apiserversource")
+	f.Setup("Create Service Account for ApiServerSource with RBAC for v1.Pod resources",
+		setupAccountAndRoleForPods(sacmName))
+
+	testNS1 := feature.MakeRandomK8sName("source-namespace")
+	testNS2 := feature.MakeRandomK8sName("source-namespace")
+	testNS3 := feature.MakeRandomK8sName("source-namespace")
```

Comment:
I'd append a fixed suffix to the prefix based on the variable name to make it more predictable and clear

Reply:
Updated